packages/mongo/src/lib/MongoProvider.ts

```diff
@@ -719,28 +728,41 @@ export class MongoProvider<StoredValue = unknown> extends JoshProvider<StoredVal
     return this._collection;
   }
 
-  private deserialize(value: string): StoredValue {
-    return new Serialize({ json: JSON.parse(value) }).toRaw<StoredValue>();
+  private _getAll(projection: { [key: string]: 1 | 0 } = { key: 1, value: 1 }) {
+    return this.collection.find<MongoProvider.DocType<StoredValue>>({}, { projection }).toArray();
   }
 
-  private serialize<Value = StoredValue>(value: StoredValue | Value) {
-    return JSON.stringify(new Serialize({ raw: value }).toJSON());
+  private _iterate(filter: Filter<MongoProvider.DocType<StoredValue>>, projection: { [key: string]: 1 | 0 } = { key: 1, value: 1 }) {
```

Comment:
```suggestion
  private iterate(filter: Filter<MongoProvider.DocType<StoredValue>>, projection: { [key: string]: 1 | 0 } = { key: 1, value: 1 }) {
```

You almost excited me, because at first I thought this was going to be an async generator.

Reply:
Aggregate returns an async iterator woo